package.json

```diff
@@ -34,12 +34,12 @@
     "mocha-junit-reporter": "^1.17.0",
     "pegin-address-verificator": "git+https://git@github.com/rsksmart/pegin-address-verifier#v0.2.1",
     "peglib": "git+https://github.com/rsksmart/rsk-peglib#v1.4.15",
+    "precompiled-fingerroot500": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-FINGERROOT",
+    "precompiled-hop400": "npm:@rsksmart/rsk-precompiled-abis@^5.0.0-HOP",
+    "precompiled-iris300": "npm:@rsksmart/rsk-precompiled-abis@^4.0.0-IRIS",
```

Comment:
why change the order here?

Reply:
I did 'npm install' and it updated it like this for some reason.